solr/core/src/java/org/apache/solr/schema/BoolField.java

```diff
@@ -97,10 +98,9 @@ public boolean incrementToken() throws IOException {
           if (done) return false;
           done = true;
           int ch = input.read();
-          if (ch==-1) return false;
+          if (ch == -1) return false;
           termAtt.copyBuffer(
-                  ((ch=='t' || ch=='T' || ch=='1') ? TRUE_TOKEN : FALSE_TOKEN)
```

Comment:
is there anything changed here?

Reply:
As i mentioned above if you select next to last in the commit view, it is easy to review, formatting is done only in the last commit